iina/fr.lproj/PrefControlViewController.strings

```diff
@@ -0,0 +1,66 @@
+
+/* Class = "NSTextFieldCell"; title = "Scroll horizontally to:"; ObjectID = "1yb-m4-1sK"; */
+"1yb-m4-1sK.title" = "Scroll horizontally to:";
+
+/* Class = "NSMenuItem"; title = "Previous / Next Media"; ObjectID = "2Ig-fw-UvW"; */
+"2Ig-fw-UvW.title" = "Média Précedent / Prochain";
+
+/* Class = "NSMenuItem"; title = "Toggle fullscreen"; ObjectID = "6F4-gm-oBg"; */
+"6F4-gm-oBg.title" = "Activer/Désactivr l'Écran Plein";
+
+/* Class = "NSTextFieldCell"; title = "Exact seek is more precise but can cause lag and high CPU usage."; ObjectID = "6FZ-Qm-0k0"; */
+"6FZ-Qm-0k0.title" = "La Recherche Exacte est plus précise, mais peut entraîner du lag et un utilisation de processeur élevé.";
```

Comment:
no need to capitalize "Recherche Exacte", and it should be : "une utilisation de processeur élevée" (le terme "utilisation" est féminin).